shared/components/events/event-timeline.tsx

```diff
@@ -37,7 +37,7 @@ const EventTimeline = ({ event }: { event: EventType }) => {
               fontSizes={{
                 cardSubtitle: '1.3rem',
                 cardText: '0.9rem',
-                cardTitle: '1.2rem',
+                cardTitle: '1.0rem',
```

Comment:
have you tested it for iphone xr?

Reply:
yes all the screens!

Reply:
![image](https://github.com/srm-kzilla/srmkzilla-net/assets/91051053/691d6411-8228-49e0-a18e-03fe4607a665)

Reply:
12 
![image](https://github.com/srm-kzilla/srmkzilla-net/assets/91051053/31874952-8f74-457f-a1da-b8e032f34765)

Reply:
<img width="1792" alt="image" src="https://github.com/srm-kzilla/srmkzilla-net/assets/74819565/c28ce755-3c2a-4b15-bafc-98ae395a643f">